libs/shared/tailwind-ui/src/lib/sonner-toaster/sonner-toaster.tsx

```diff
@@ -19,7 +19,11 @@ const Toaster = ({ ...props }: ToasterProps) => {
 					actionButton:
```

Comment:
Sorry, which states, which view, why?

Reply:
It's not one view, it's for notification system in case we show some message. 
Currently it's in monochrome tone so added colours

Commit message improved

<img width="637" alt="Screenshot 2024-01-15 at 05 59 47" src="https://github.com/ts-factory/bublik-ui/assets/156439139/b997a8fe-aa2a-418b-a83d-d4d2afc59691">